ci/docs.go

```diff
@@ -15,11 +15,11 @@ type Docs struct {
 
 const (
 	generatedSchemaPath = "docs/docs-graphql/schema.graphqls"
-	generatedCliZenPath = "docs/current_docs/reference/979596-cli.mdx"
+	generatedCliZenPath = "docs/current_docs/reference/cli.mdx"
```

Comment:
I think the CI test is failing because of this change, but I'm not sure how to fix it.